### PR TITLE
Fixed issue #11383

### PR DESCRIPTION
**Pull request by sergign60**

### CODE REVIEW

**Comment by sergign60** [comment]
CC: @Dmitri-Botcharnikov @lucenticus

**Comment by sergign60** [comment]
https://github.com/dotnet/coreclr/issues/11383

File: src/vm/eventtrace.cpp [hunk 1]
```diff
@@ -6760,9 +6762,9 @@ VOID ETW::MethodLog::SendHelperEvent(ULONGLONG ullHelperStartAddress, ULONG ulHe
                                      ulHelperSize, 
                                      0, 
                                      methodFlags, 
-                                     NULL, 
+                                     W16_NULLSTRING,
```
Comment (brianrob) [inline src/vm/eventtrace.cpp]: Can you tell me more about why NULL is not an OK value here?

Comment (sergign60) [inline src/vm/eventtrace.cpp]: @brianrob I described the problem in #11383

Comment (ramarag) [inline src/vm/eventtrace.cpp]: i am not entirely sure if this is necessary will leave it to @brianrob to make the call

**Comment by ramarag** [comment]
LGTM, @leculver do PTAL

**Comment by leculver** [comment]
LGTM as well.

File: src/vm/eventtrace.cpp (hunk 1 above)
Comment (brianrob) [inline src/vm/eventtrace.cpp]: @sergig60, my concern is that there are likely to be many call sites that pass NULL for strings.  Your fix already ensures that if a NULL str is passed that it short circuits and never calls memcpy.  Thus, I don't think you need to pass an empty string.

Comment (sergign60) [inline src/vm/eventtrace.cpp]: @brianrob Ok, If you are sure, I'll return NULL's

**Pull request merged**